### PR TITLE
First assignment

### PR DESCRIPTION
**Pull request by celestinekakai**

### CODE REVIEW

File: assignment1.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// First assignment:
```
Comment (M-Shehu) [inline assignment1.js:1]: Nice comments

**Comment by celestinekakai** [comment]
@M-Shehu requesting review for assignment1

File: assignment1.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+// Create a function that takes in 2 arguments which are numbers. 
+// If the first argument is greater than the 2nd, it should return a string 'First number is greater than second number'. 
+// Else if the second argument is greater than the 1st, it should return a string 'Second number is greater than the first number'. 
+// Else it should return a string 'The numbers are the same'
+
+const compareNumbers = (number1, number2) => {
+    if (number1 > number2) {
+        return 'First number is greater than second number'; }
```
Comment (M-Shehu) [inline assignment1.js:9]: The semicolon ending this line needs to be on the next line before the “else” keyword

File: assignment1.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    else if (number2 > number1) { 
+        return 'Second number is greater than the first number';
+        }
```
Comment (M-Shehu) [inline assignment1.js:12]: Similar to earlier, the semicolon on this line needs to be on the next before the “else” keyword

Comment (celestinekakai) [reply assignment1.js:12]: Interesting, where did you pick up these linting errors? You can show me during sync.

**Pull request merged**